packages/wonder-blocks-tooltip/src/components/tooltip-popper.tsx

```diff
@@ -172,21 +223,39 @@ export default class TooltipPopper extends React.Component<Props> {
     }
 
     render(): React.ReactNode {
-        const {anchorElement, placement} = this.props;
+        const {anchorElement, placement, rootBoundary} = this.props;
+
+        const smallViewportModifier: SmallViewportModifier = {
+            name: "smallViewport",
+            enabled: true,
+            phase: "main",
+            fn: _modifyPosition,
+        };
+
+        const modifiers: Modifiers[] = [smallViewportModifier];
```

Comment:
**suggestion:** Could you please add a TODO comment here pointing to the floating-ui Jira ticket? this would be helpful for us to track this change in the future.

```suggestion
        // TODO(WB-1680): Use floating-ui's 
        const modifiers: Modifiers[] = [smallViewportModifier];
```